Userland/Libraries/LibJS/Tests/builtins/Array/Array.prototype.sort.js

```diff
@@ -153,7 +153,7 @@ describe("Array.prototype.sort", () => {
 
         obj = { 0: 3, 1: 2, 2: 1, a: "b", hello: "friends!", length: 2 };
         expect(Array.prototype.sort.call(obj)).toBe(obj);
-        expect(obj).toEqual({ 0: 2, 1: 3, 2: 1, 3: 0, a: "b", hello: "friends!", length: 2 });
+        expect(obj).toEqual({ 0: 2, 1: 3, 2: 1, a: "b", hello: "friends!", length: 2 });
```

Comment:
This commit's changes should be moved into the commit that changes the deep-equals function, so that the test continues to pass.

Reply:
If we want changes that fix broken tests to be in the same commit that broke the tests, should I only have one commit in this case?

Reply:
If your other commits are all fixing fixing tests that the first commit broke, then yes.

Reply:
Done